client/components/Parts/Dialogs/Input.tsx

```diff
@@ -14,10 +15,23 @@ export interface InputDialogProps extends Omit<ConfirmDialogProps, 'onConfirm'>
 export default function InputDialog(props: InputDialogProps) {
   const { value: initValue, prefix, placholder, spellcheck, onConfirm, ...rest } = props
   const [value, setValue] = React.useState(initValue || '')
-  const handleConfirm = () => onConfirm && onConfirm(value)
+  const [textFieldError, setTextFieldError] = React.useState(false)
+  const [errorHelperText, setErrorHelperText] = React.useState('')
+  const handleConfirm = () => {
+    if(value !== '' && helpers.isUrlValid(value)){
+      setTextFieldError(false)
+      return onConfirm && onConfirm(value)
+    } else {
+      setTextFieldError(true)
+      if(value === '') { setErrorHelperText('Empty field!') }
```

Comment:
I think we should add a more descriptive message. This will fail not only on empty URLs but also malformed ones.

Any suggestion @romicolman ? I'm thinking on something like: `This field should contain a valid URL starting with http or https.`

Reply:
Let's add this message: "Please, enter a valid URL". If it's large, "Enter a valid URL".  Regarding the URL question. People most of the time  copy and past from the browser, so we are OK.

Reply:
@guergana 
I think it might be a rabbit hole. In the end, a real validation happens the server side whether the app can load and parse the file or not. And handling it is this one - https://github.com/okfn/opendataeditor/issues/295